README.md

```diff
@@ -15,8 +15,7 @@ go get -u -v github.com/spiffe/go-spiffe
 
 ## Importing it in your Go code
 
-See examples in [the tests](./spiffetest), [the examples section](./examples)
-or visit the [GoDoc](https://godoc.org/github.com/spiffe/go-spiffe) for more information
+See the [examples](./examples) or visit the [GoDoc](https://godoc.org/github.com/spiffe/go-spiffe) for more information.
```

Comment:
```suggestion
See the [examples](./examples) or visit the [GoDoc](https://pkg.go.dev/github.com/spiffe/go-spiffe) for more information.
```

Reply:
Good call. I'm going to fold this into #117. After that one is merged we can tag the new release for v1.0.0 or something. Otherwise pkg.go.dev picks up the wrong commit (because we have no v1 tag)